### PR TITLE
Multiple/queries

### PR DESCRIPTION
**Pull request by Theodoraldo**
**At this time, I :** 

- use primary key & foreign key mechanism for joining tables.

- query multiple tables.

- prepare complex queries that answer analytical questions.

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -13,4 +13,32 @@ VALUES ('Charmander', '2020-02-08', 0, 'f', -11.00),
...
+('Dean Winchester', 14),
+('Jodie Whittaker', 38);
+	
+INSERT INTO species (name) VALUES
+	('Pokemon'),
+	('Digimon');
+
+UPDATE animals SET species_id = (SELECT id FROM species WHERE name = 'Digimon') WHERE name LIKE '%mon';
+
+UPDATE animals SET species_id = (SELECT id FROM species WHERE name = 'Pokemon') WHERE name NOT LIKE '%mon';
```
Comment (Olamarx) [inline data.sql]: - [ ] This should be where name is null not where name is not like %mon.

File: queries.sql [hunk 2]
```diff
@@ -82,4 +82,20 @@ SELECT MAX(weight_kg), species FROM animals GROUP BY species;
...
+SELECT AVG(escape_attempts), species FROM animals WHERE DATE_PART('YEAR',date_of_birth) BETWEEN 1990 AND 2000 GROUP BY species;
+-- -----------------------------------------------------------------------------------------------------
+
+SELECT ani.name, own.full_name FROM animals ani INNER JOIN owners own ON ani.owner_id = own.id WHERE own.full_name IN ('Melody Pond');
+
+SELECT ani.name animal_name, sp.name type_name FROM animals ani INNER JOIN species sp ON ani.species_id = sp.id WHERE sp.name = 'Pokemon';
+
+SELECT ani.name animal_name, own.full_name owner_name FROM animals ani FULL JOIN owners own ON ani.owner_id = own.id;
+
+SELECT COUNT(*) FROM animals GROUP BY species_id;
```
Comment (Olamarx) [inline queries.sql]: - [ ] I will encourage you to use join in querying this data too as required.

File: queries.sql [hunk 3]
```diff
@@ -82,4 +82,20 @@ SELECT MAX(weight_kg), species FROM animals GROUP BY species;
...
+
+SELECT ani.name animal_name, sp.name specie_name FROM animals ani JOIN species sp ON ani.species_id = sp.id INNER JOIN owners own ON ani.owner_id = own.id WHERE sp.name = 'Digimon' AND own.full_name = 'Jennifer Orwell';
+
+SELECT * FROM animals WHERE owner_id = (SELECT id FROM owners WHERE full_name = 'Dean Winchester') AND escape_attempts = 0;
```
Comment (Olamarx) [inline queries.sql]: - [ ] I will encourage you to use join here too.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @Theodoraldo, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️
- Readme file professional ✔️
- All queries made ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Theodoraldo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**